cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -1,17 +1,44 @@
 #!/bin/sh
 
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ -z "${AWS_SHARED_CREDENTIALS_FILE}" ]; then echo "ENV VAR AWS_SHARED_CREDENTIALS_FILE is not set"; exit 1; fi
```

Comment:
nit: prefer

```sh
if [[ -z "${AWS_SHARED_CREDENTIALS_FILE:-}" ]]; then
    echo "Required variable \$AWS_SHARED_CREDENTIALS_FILE unset"
    exit 1
fi
```

Reply:
Also, can't we just omit this? The AWS library will fail when no credentials are found, and this script could be used locally with the credentials in `$HOME`

Reply:
Deleted.